extra-dev/packages/coq-elpi/coq-elpi.1.12.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name: "coq-elpi"
+version: "dev"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" ]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/LPCIC/coq-elpi"
+bug-reports: "https://github.com/LPCIC/coq-elpi/issues"
+dev-repo: "git+https://github.com/LPCIC/coq-elpi"
+
+build: [ [ make "build"   "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" "OCAMLWARN=" ]
+         [ make "test"    "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ] {with-test}
+       ]
+install: [ make "install" "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ]
+depends: [
+  "stdlib-shims"
+  "ocaml" {>= "4.07"}
+  "elpi" {>= "1.13.6" & < "1.14.0~"}
+  "coq" {>= "8.15" & < "8.16~" }
+]
+tags: [ "logpath:elpi" ]
+synopsis: "Elpi extension language for Coq"
+description: """
+Coq-elpi provides a Coq plugin that embeds ELPI.
+It also provides a way to embed Coq's terms into λProlog using
+the Higher-Order Abstract Syntax approach
+and a way to read terms back.  In addition to that it exports to ELPI a
+set of Coq's primitives, e.g. printing a message, accessing the
+environment of theorems and data types, defining a new constant and so on.
+For convenience it also provides a quotation and anti-quotation for Coq's
+syntax in λProlog.  E.g. `{{nat}}` is expanded to the type name of natural
+numbers, or `{{A -> B}}` to the representation of a product by unfolding
+ the `->` notation. Finally it provides a way to define new vernacular commands
+and
+new tactics."""
+url {
+src: "https://github.com/LPCIC/coq-elpi/archive/v1.12.0.tar.gz"
+checksum: "sha256=a791a536a163d6c9625eb32ad4ed4ad890667d6293f74190b62e14a73a73f0a3"
```

Comment:
```suggestion
checksum: "sha256=796225b6a569ca47ca2a2d6a7c15d1d57de971ef669bde7dd6321ce34f61fa29"
```

Reply:
thanks, I did forget to pass curl a flag :-/